node/pkg/bus/types.go

```diff
@@ -5,11 +5,11 @@ const (
 	// 1. admin: works as user interface to control whole system, sends message to other packages to control
 	// 2. fetcher: fetches data from different sources and stores in db, sends message to aggregator in case of deviation
 	// 3. aggregator: aggregates data through data sent from other nodes and stores global aggregates into db
-	// 4. submitter: submits price into blockchain in regular basis
+	// 4. reporter: submits price into blockchain in regular basis
 	ADMIN      = "admin"
 	FETCHER    = "fetcher"
 	AGGREGATOR = "aggregator"
-	SUBMITTER  = "submitter"
+	REPORTER   = "reporter"
```

Comment:
Nice! Having same terminology will help people to compare old and new version of aggregator.